src/librustc_middle/ty/print/pretty.rs

```diff
@@ -488,7 +487,7 @@ pub trait PrettyPrinter<'tcx>:
             }
             ty::Ref(r, ty, mutbl) => {
                 p!(write("&"));
-                if self.region_should_not_be_omitted(r) {
+                if self.region_should_be_omitted(r) {
                     p!(print(r), write(" "));
```

Comment:
The region is printed if it should be omitted/left out?

Reply:
Argh, you're totally right, that's what I get for working so late - dropped that commit.